frontend/src/routes/(authenticated)/project/[project_code]/AddProjectMember.svelte

```diff
@@ -23,15 +23,15 @@
     const { response, formState } = await formModal.open(async () => {
       const { error } = await _addProjectMember({
         projectId,
-        userEmail: $form.email,
+        usernameOrEmail: $form.usernameOrEmail,
         role: $form.role,
       });
 
       if (error?.byType('NotFoundError')) {
-        return { email: [$t('project_page.add_user.project_not_found')] };
+        return { usernameOrEmail: [$t('project_page.add_user.project_not_found')] };
```

Comment:
I wonder why the message string supplied by the server is getting turned into "Unknown error" at some point in the GQL machinery? If it was preserved, we could differentiate "Project not found" vs "User not found". But since "Project not found" can only happen if one admin loads the project page, then a second admin deletes the project, then the first admin submits the add member form without reloading the page... yeah, that's pretty much never going to happen. So I'm fine with option 2, where we choose a different exception and put in a comment explaining why that exception is used for "project not found".

Reply:
Option 2 implemented in commit 64fd5789.

Reply:
Oh, I was wrong. You actually can pull out the message. I tried it before, but I must have messed up somehow.
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/4b169228-837f-4991-b82c-4fd970832116)

That might be the cleaner way to do this, but I'm fine with the way it is.
Resolve this as you like.

Reply:
Done in commit 262cddb4ce634529fe9f3868c8b389001bfd0667, where I went back to NotFound for projects and users and used the message to distinguish them in the UI. Which is the cleaner way to do it, and the way I wanted to do it in the first place.